code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -642,7 +641,7 @@
 
 	if(!sheet_reagents[O.type] && (!O.reagents || !O.reagents.total_volume))
 		user << "\The [O] is not suitable for blending."
-		return 1
+		return 0

```

Comment:
The returned value determines whether or not the interaction is 'finished'. If a 1 is returned, the attack terminates and does not call afterattack, which is where the gripper does most of its work